configuration/exodus-pipeline.yaml

```diff
@@ -273,25 +283,26 @@ Resources:
   CodePipelineCloudTrail:
     Type: AWS::CloudTrail::Trail
     # Create trail for prod pipeline
-    Condition: IsProd
+    Condition: IsExodusProd
     Properties:
-      S3BucketName: exodus-pipeline-artifacts
+      S3BucketName: cloudtrail-logs
       EventSelectors:
         - DataResources:
           - Type: AWS::S3::Object
             Values:
-              - arn:aws:s3:::exodus-pipeline-artifacts/build-stage
+              - "arn:aws:s3:::"
           ReadWriteType: WriteOnly
+          IncludeManagementEvents: false
       IncludeGlobalServiceEvents: true
       IsLogging: true
-      IsMultiRegionTrail: true
```

Comment:
This got me curious and, after looking into it, I don't think we need IsLogging or IncludGlobalServiceEvents either.

Reply:
Hmm, I am not sure about IsLogging. I thought IsLogging defines whether the trail is enabled or disabled (see each trail's "status" field [here](https://console.aws.amazon.com/cloudtrail/home?region=us-east-1#/dashboard), or the on/off switch within a Trail). What is your understanding of this bool? Either way, +1 to removing IncludeGlobalServiceEvents.

Reply:
Oh, I think you're right about IsLogging. I misread the doc for it.